package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cardano-hw-cli",
   "version": "1.12.0",
-  "commit": "081c692dc9eaff4708c3b0cfa84a27815bd556df",
+  "commit": "c9bcf79f4b1053aa1af1cbaf3b602e3ed1879815",
```

Comment:
first time I see this property in `package.json` - do you know why it's here are what is it's purpose?

Reply:
No. I think it appeared when we replaced nexe with pkg. It gets automatically updated during build.
@davidmisiak Any idea why it is there? I'd rather remove it, I see no point for its existence for our case.

Reply:
The release build of the tool is able to print the commit hash when running `cardano-hw-cli --version`, and it reads it from here. Before nexe->pkg migration, the build script copied the package.json to the build folder and added the commit hash only to the copied file, but I think that this is no longer possible / difficult with pkg for some reason. @gabrielKerekes might remember the details, as he was doing the migration.

I don't have a strong opinion on this, if we want to print the commit hash, we have to store it somewhere and this look acceptable (though unusual) to me.